en/docs/tutorials/construct-statements.md

```diff
@@ -0,0 +1,119 @@
+# Construct Statements
+
+The Statement Editor in the Choreo Web Editor is a tool that allows you to construct Ballerina statements in your Choreo Editor even without an in-depth knowledge of the Ballerina Programming Language.
+
+It provides the following:
+
+- Context-based suggestions
+
+- Ballerina libraries (standard libraries and language libraries)
+
+- Pre-defined expression templates
+
+- Documentation for functions and methods with the support to easily configure parameters
+
+This guide walks you through the steps to build the following statement using the Statement-Editor.
+
+```
+string maxValue = "Max : " + int0:max(30, -89).toString();
+```
+
+The above is a variable statement named `maxValue` that derives the maximum value in a list of two integers and passes the value in string format (for example, as `"Max : 30"` ).
+
+To build the statement, follow these steps:
```

Comment:
We should perhaps mention that the steps on starting with the Echo template and so on are just for the purpose of this tutorial to get to the statement Editor.

Reply:
Moved these steps to an info note.